src/domain/individualHarbor/harborProperties/HarborProperties.tsx

```diff
@@ -65,9 +72,28 @@ const HarborProperties: React.SFC<HarborPropertiesProps> = ({
               </Section>
               <Section>
                 <ExternalLink href={serviceMapUrl} variant="withArrow">
-                  {t('harbors.details.serviceMap')}
+                  {t('common.terminology.serviceMap')}
                 </ExternalLink>
               </Section>
+              {maps.length > 0 && (
+                <Section>
+                  {maps.length > 1 ? (
```

Comment:
We could create a reusable component `HarborMaps` that would be used here and in the details.

Reply:
I created one called `HarborMapLinks`, it's under `cards`